### PR TITLE
Add changelog for 0.7.0

### PR DESCRIPTION
**Pull request by akien-mga**

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -6,6 +6,104 @@ OpenDungeons since version 0.4.7, in reverse chronological order.
...
+
+**Highlights:**
+
+* Multiplayer lobby using a master server, to find other players all over the world
+* Animated main menu with scripted events!
+* Play voice-overs on some game events
+* Vastly improved creatures behaviour, including support for spell casting,
+  elemental damage, better ranged behaviour, and many customisation features
+* New rooms: Arena, Bridges, Casino, Prison, Torture room
+* New creatures: Cultist, Elf, Dark Elf, Lich, Nature Monster, Rat, Roach, Skeleton
```
Comment (Loobinex) [inline RELEASE-NOTES.md:19]: No monk or lava spawn from PR #1190?

Comment (akien-mga) [inline RELEASE-NOTES.md:19]: Ah right, I added them in the graphics section but not in the highlights. Will add them when putting the release data, or if other stuff has to be added before.

Comment (hwoarangmy) [inline RELEASE-NOTES.md:19]: wow, @Loobinex do not miss anything ^^

**Comment by hwoarangmy** [comment]
ok, so let's merge

**Pull request merged**